internal/receiver/scriptreceiver/README.md

```diff
@@ -0,0 +1,84 @@
+# Script Receiver
```

Comment:
Thanks for sharing more info. What about something like Scripted Inputs Receiver (`scripted_inputs`) or similar to match the existing offering?